packages/web/src/types.ts

```diff
@@ -13,19 +13,18 @@ export type Mode = 'auto' | 'development' | 'production';
 export type AllowedPropertyValues = string | number | boolean | null;
 
 export type BeforeSend = (event: BeforeSendEvent) => BeforeSendEvent | null;
+
 export interface AnalyticsProps {
   beforeSend?: BeforeSend;
   debug?: boolean;
   mode?: Mode;
-  route?: string | null;
```

Comment:
**question** do we not need that here anymore?

Reply:
We don't need it inside the props for `inject()`, nor the props of framework-specific components.

It used to be in props for `inject()`, but the function never used them.